### PR TITLE
add assignments

### PR DESCRIPTION
**Pull request by crhistianramirez**

### CODE REVIEW

File: src/pages/docs/main-concepts/2. assignments.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Now that we have an understanding of the various ordercloud parties (companies and groups) we can begin to talk about assignments. When you save an assignment you are creating a relationship between a party and an object (ordercloud "thing" such as product, category, etc). For example you might assign a usergroup to a category, thereby granting a user in that group visibility to that category.
+
+There are basic principles around how assignments work that are critical to understanding our data model and more importantly how the data model can be applied to solve the most complex ordering scenarios efficiently
+- Assignments are used to define a relationship
+- Assignments are inclusive
+- Assignments can be made at different levels
+- Assignments cascade down higher levels to the individual user
+- Assignments are many to many
+
+## Assignments are inclusive
```
Comment (amrarick26) [inline src/pages/docs/main-concepts/2. assignments.md]: Let's standardize title casing here - "Assignments are Inclusive"

File: src/pages/docs/main-concepts/3. me-resource.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+For example let's say your buyer user is assigned an address at the buyer level, and another address at the usergroup level. Instead of listing assignments between the user and both those levels and manually resolving the addresses yourself, you can simply make a list call as that buyer user to [me/addresses](TODO:link-to-me-addresses) and quickly get the assigned addresses in one quick and fluid motion.
+
+## Ability to admin personal resources
+In addition to being able to quickly get information about the currently authenticated user, the Me resource provides a set of admin endpoints for personal resources (Credit Cards and Addresses) that only that user can see. Not even admins have access to viewing the data created by these users without something like [impersonation](TODO:link-to-impersonation-feature).
+
+
+## Overview
+We've seen how assignments can be made and flow down to the individual user, and how easy it is by using the Me perspective to get those assignments. In the next section we're going to talk about Security Profiles which let you lock down which endpoints a user can access.
```
Comment (amrarick26) [inline src/pages/docs/main-concepts/3. me-resource.md]: Should we use the same syntax here for `Me` like you used in the second paragraph?

File: src/pages/docs/main-concepts/6. product-visibility.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+[TODO:link-to-image]
+
+
+It might be intuitive to see price as a property of a product but by decoupling any price related things (price schedule) from the product we can represent the same product at different prices to different users. Let's say we want that same product to be have a different price for a different user. No problem!
+
+[TODO:image-with-sameproduct-two-prices]
+
+
+Hopefully you can begin seeing the power and flexibility we gain by using assignments. By the way, unique pricing is really just skimming the surface of what we can do with priceschedules. A price schedule can do things like set min or max quantity, restrict quantity to set intervals, and even track inventory!
```
Comment (amrarick26) [inline src/pages/docs/main-concepts/6. product-visibility.md]: Price schedules or priceschedules?

File: src/pages/docs/main-concepts/2. assignments.md (hunk 1 above)
Comment (crhistianramirez) [reply src/pages/docs/main-concepts/2. assignments.md]: How would this one look with the standardized casing?
Assignments can be made at different levels

Assignments can be Made at Different Levels ?

Comment (amrarick26) [reply src/pages/docs/main-concepts/2. assignments.md]: Yeah I believe so

**Pull request merged**